api/memory/graphiti_tool.py

```
@@ -255,34 +181,97 @@ async def _ensure_entity_nodes_direct(self, user_id: str, database_name: str) ->
+                ### Inputs
+                1. Existing user summary (overall + personal info):
+                {summary}
+
+                2. Latest Q&A conversational memory:
+                {conv_text}
+
+                ### Task
+                - Produce a new user summary of his overall preferences and his personal information.
+                - *Important*: Ensure that the summary is contain any personal statements or preferences expressed by the user.
+                - Preserve existing personal information, preferences, and tendencies from the old summary.
+                - Integrate any **new insights** about the user’s interests, behaviors, or database usage patterns from the latest memory.
```

Ruff (0.12.2)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)